### PR TITLE
front: import opendata: correct data format

### LINKED ISSUE #4768: Importing train schedules doesn't work
### What happened?

A problem appears when trying to import train schedules. 

### What did you expect to happen?

Train schedules should be well displayed.

### How can we reproduce it (as minimally and precisely as possible)?

- Go to scenario
- Click on "import"
- After selecting an origin and a destination, click on the "search" icon
- A toast saying "Impossible de convertir les données en TrainSchedule" appears after few seconds (It should be translated too).

### What operating system, browser and environment are you using?

- **Browser**: Firefox v116
- **Env**: Dev

### OSRD version (top right corner `Account` button > `Informations`)

8684ff50

### PR DESCRIPTION
**Pull request by nicolaswurtz**

### CODE REVIEW

File: front/public/locales/en/operationalStudies/importTrainSchedule.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "errorNoFrom": "You must fill in an origin.",
     "errorNoTo": "You must fill in a destination.",
     "errorSameFromTo": "Origin and destination must be different",
+    "errorImport": "Data conversion failed",
```
Comment (clarani) [inline front/public/locales/en/operationalStudies/importTrainSchedule.json]: This translation should be higher (alphabetical order) :face_with_peeking_eye:

File: front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx [hunk 2]
```diff
@@ -91,28 +91,21 @@ export default function ImportTrainScheduleConfig({
         return true;
       }
       const hasInvalidteps = trainSchedule.steps.some((step) =>
```
Comment (clarani) [inline front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx]: Oups, typo, could you correct it please ? :pray: 
```suggestion
      const hasInvalidSteps = trainSchedule.steps.some((step) =>
```

**Review by clarani — APPROVED** [review]
LGTM, tested in local and the import works well :heavy_check_mark:

File: front/public/locales/en/operationalStudies/importTrainSchedule.json (hunk 1 above)
Comment (nicolaswurtz) [reply front/public/locales/en/operationalStudies/importTrainSchedule.json]: Yes !

File: front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx (hunk 2 above)
Comment (nicolaswurtz) [reply front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx]: Yes, corrected ;)

**Pull request merged**